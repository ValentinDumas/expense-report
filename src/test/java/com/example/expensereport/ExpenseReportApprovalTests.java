package com.example.expensereport;

public class ExpenseReportApprovalTests {

}
