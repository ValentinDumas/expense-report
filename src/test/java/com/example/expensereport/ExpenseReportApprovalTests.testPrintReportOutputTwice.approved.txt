Expenses <DATE>
Dinner	4000	 
Breakfast	800	 
Car Rental	15000	 
Meal expenses: 4800
Total expenses: 19800
Expenses <DATE>
Dinner	4000	 
Breakfast	800	 
Car Rental	15000	 
Meal expenses: 4800
Total expenses: 19800
